modules/candidate_list/jsx/columnFormatter.js

```diff
@@ -8,6 +8,10 @@
  * @return {*} a formated table cell for a given column
  */
 function formatColumn(column, cell, rowData) {
+  if (column === 'Visits') {
```

Comment:
Other module use  this 
`
  if (loris.hiddenHeaders.indexOf(column) > -1) {
    return null;
  }
`
Unless you are trying to remove loris.hiddenHeaders

Reply:
I can change this.. but the loris.hiddenHeaders doesn't make sense to me. I don't understand why it's a global variable and not a prop.